src/ontology/cl-edit.owl

```diff
@@ -4843,7 +4843,7 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "Clara cell")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "bronchiolar non-ciliated cell")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "club cell of bronchiole")
 AnnotationAssertion(rdfs:label obo:CL_0000158 "club cell")
-EquivalentClasses(obo:CL_0000158 ObjectIntersectionOf(obo:CL_0002328 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0001789) ObjectSomeValuesFrom(obo:RO_0002104 obo:GO_0005902) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0032940) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0043129)))
+EquivalentClasses(obo:CL_0000158 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051) ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0001789) ObjectSomeValuesFrom(obo:RO_0002104 obo:GO_0005902) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0043129)))
```

Comment:
This cell type has been noted in various locations along the tracheobronchial tree. 
See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7528681/figure/F2/

Consider converting the equivalence axiom into subClass axioms.

Reply:
It looks like it varies [depending on the species](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6290947/):
![image](https://user-images.githubusercontent.com/94959119/231818482-7d35c109-2a0c-481c-b36b-c89f4db01b5c.png)